src/webgpu/capability_info.ts

```diff
@@ -70,8 +70,8 @@ export const kBufferUsages = numericKeysOf<GPUBufferUsage>(kBufferUsageInfo);
 
 // Textures
 
-export const kRegularTextureFormatInfo = /* prettier-ignore */ makeTable(
-                           ['renderable', 'multisample', 'color', 'depth', 'stencil', 'storage', 'copySrc', 'copyDst', 'bytesPerBlock', 'blockWidth', 'blockHeight',              'extension'] as const,
+const kTexFmtInfoHeader =  ['renderable', 'multisample', 'color', 'depth', 'stencil', 'storage', 'copySrc', 'copyDst', 'bytesPerBlock', 'blockWidth', 'blockHeight',              'extension'] as const;
```

Comment:
I think it is better to move the template of kTexFmtInfoHeader to the top. But `grunt pre` fail because I don't want to run `grunt fix`, the latter will re-layout and make all parameters of kTexFmtInfoHeader on a separate line. I run `grunt pre --force`. It can pass.

Reply:
I actually just considered but then didn't do this in #486. Reasoning was that having the header multiple times makes it easier to read. But I am also totally fine with this change; whatever you think.

The lint fix is to use `/* prettier-ignore */` on the previous line, see [1]

Reply:
What a coincidence... Hmm, I agree that the current one is more friendly for reading info from the table (and reading is more frequently being used than writing, just as you and Austin discussed). So I changed back, but added a comment to explain it.